examples/ASDKgram/Sample/PhotoFeedNodeController.m

```diff
@@ -93,30 +93,30 @@ - (void)loadPageWithContext:(ASBatchContext *)context
   [_photoFeed requestPageWithCompletionBlock:^(NSArray *newPhotos){
     
     [self insertNewRowsInTableView:newPhotos];
-    [self requestCommentsForPhotos:newPhotos];
+//    [self requestCommentsForPhotos:newPhotos];
     if (context) {
       [context completeBatchFetching:YES];
     }
   } numResultsToReturn:20];
 }
 
-- (void)requestCommentsForPhotos:(NSArray *)newPhotos
-{
-  for (PhotoModel *photo in newPhotos) {
-    [photo.commentFeed refreshFeedWithCompletionBlock:^(NSArray *newComments) {
-      
-      NSInteger rowNum      = [_photoFeed indexOfPhotoModel:photo];
-      NSIndexPath *cellPath = [NSIndexPath indexPathForRow:rowNum inSection:0];
-      PhotoCellNode *cell   = (PhotoCellNode *)[_tableNode.view nodeForRowAtIndexPath:cellPath];
-      
-      if (cell) {
-        [cell loadCommentsForPhoto:photo];
-        [_tableNode.view beginUpdates];
-        [_tableNode.view endUpdates];
-      }
-    }];
-  }
-}
+//- (void)requestCommentsForPhotos:(NSArray *)newPhotos
```

Comment:
Actually, I mean that the comment is required in the Apple framework version of this file. This code should be deleted from this file as it should no longer be necessary with the more efficient handling of this at the cell level.